ports/yasm-tool/vcpkg-port-config.cmake

```
@@ -0,0 +1,31 @@
+set(Z_YASM_TOOL_HELPER_LIST_DIR "${CMAKE_CURRENT_LIST_DIR}")
+
+function(yasm_tool_helper)
+    cmake_parse_arguments(PARSE_ARGV 0 arg
+        "APPEND_TO_PATH;PREPEND_TO_PATH"
+        "OUT_VAR"
+        ""
+    )
+
+    if(DEFINED arg_UNPARSED_ARGUMENTS)
+        message(FATAL_ERROR "Unexpected arguments to yasm_tool_helper(): ${arg_UNPARSED_ARGUMENTS}")
+    endif()
+
+    if(CMAKE_HOST_WIN32)
```

```suggestion
    if(@BUILT_EXE@)
```
This script should be embedded with whether the executable was built.